### PR TITLE
LPS-80577 Single AlloyEditor

### PR DESCRIPTION
**Pull request by p2kmgcl**
/cc @ealonso

Esta va despues de esta: https://github.com/jbalsas/liferay-portal/pull/1202
Es para que solo haya un alloyeditor a la vez en el editor de page templates

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test

**Comment by p2kmgcl** [comment]
(Rebaseado sobre master, ya no hay conflictos)

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/FragmentEntryLink.es.js [hunk 1]
```diff
@@ -283,14 +292,23 @@ class FragmentEntryLink extends Component {
...
 	 * @private
 	 * @review
 	 */
 
-	_updateTranslationStatus(editableId, value) {
+	_updateEditableStatus(editableId, value, defaultValue, mappedField) {
 		const element = this.element.querySelector(`lfr-editable[id="${editableId}"]`);
 
-		element.classList.remove('translated', 'untranslated');
+		element.classList.remove(
```
Comment (jbalsas) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/FragmentEntryLink.es.js]: There's no support for this on IE11 -> https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Browser_compatibility

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/FragmentEntryLink.es.js [hunk 2]
```diff
@@ -301,15 +281,20 @@ class FragmentEntryLink extends Component {
...
 		const element = this.element.querySelector(`lfr-editable[id="${editableId}"]`);
 
-		element.classList.remove(
-			'mapped',
-			'translated',
-			'unmapped',
-			'untranslated'
-		);
+		if (element) {
+			element.classList.remove(
```
Comment (jbalsas) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/FragmentEntryLink.es.js]: There's no support for this on IE11 -> https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Browser_compatibility

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/fragment_processors/EditableTextFragmentProcessor.es.js [hunk 3]
```diff
@@ -109,54 +139,36 @@ class EditableTextFragmentProcessor {
...
-
-				editor.destroy();
-				editableField.innerHTML = editor.get('nativeEditor').getData();
-			}
-		);
-
-		this._editors = [];
+		if (editableField !== this._editableField) {
+			this._destroyEditor();
+			this._createEditor(editableField);
```
Comment (jbalsas) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/fragment_processors/EditableTextFragmentProcessor.es.js]: Add a line here, just to make sure we represent that order matters in these two lines

**Comment by jbalsas** [comment]
I'm not sure about the last portion, the setting the single AlloyEditor bit... I'm seeing a weird behaviour on this. See gif:

updates

I'll push the other bit first, and we can verify the editor part on wednesday.

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/p2kmgcl/liferay-portal/pull/98. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/83ed18f2e332~1...83ed18f2e332) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**